docs/graphql/manual/deployment/downgrading.rst

```diff
@@ -35,1276 +35,27 @@ During upgrades the server automatically migrates the catalogue to the latest ve
-
-+-----------------------------------+---------------------+
-| Hasura GraphQL engine versions    | Catalogue version   |
-+===================================+=====================+
-| v1.0.0-beta.10                    | 27                  |
-+-----------------------------------+---------------------+
-| v1.0.0-beta.8 - v1.0.0-beta.9     | 26                  |
-+-----------------------------------+---------------------+
-| v1.0.0-beta.7                     | 24                  |
-+-----------------------------------+---------------------+
-| v1.0.0-beta.6                     | 22                  |
-+-----------------------------------+---------------------+
-| v1.0.0-beta.4 - v1.0.0-beta.5     | 19                  |
-+-----------------------------------+---------------------+
-| v1.0.0-beta.2 - v1.0.0-beta.3     | 17                  |
-+-----------------------------------+---------------------+
-| v1.0.0-beta.1                     | 16                  |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha42 - v1.0.0-alpha45   | 13                  |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha41                    | 12                  |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha40                    | 11                  |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha39                    | 10                  |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha36 - v1.0.0-alpha38   | 9                   |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha34 - v1.0.0-alpha35   | 7                   |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha31 - v1.0.0-alpha33   | 6                   |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha30                    | 5                   |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha29                    | 4                   |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha21 - v1.0.0-alpha28   | 3                   |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha16 - v1.0.0-alpha20   | 2                   |
-+-----------------------------------+---------------------+
-| v1.0.0-alpha0 - v1.0.0-alpha15    | 1                   |
-+-----------------------------------+---------------------+
-
 Downgrading across catalogue versions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can downgrade the catalogue from a particular version to its previous version by executing the appropriate SQL statements provided below.
+You can downgrade the catalogue from a particular version to a previous version by executing the ``graphql-engine`` executable on the command line,
+with the ``downgrade`` command, specifying the desired catalog version using one of the ``--to-`` flags.
 
 .. note::
```

Comment:
This seems like a rather long note. How about just having the third point as a note (as it's the unexpected case) and have the rest part of the normal tutorial?

Reply:
Done, thanks.